ionos_sbx/marketplace/iam/apisix/templates/eurodyn-apikey-secret-sealed-secret.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Please squash the commits and prefix commit name with environment name 'sbx'

Reply:
Hi, I did a rebase, but there is no squash option, do i have the rights? If not, please give me access. Thanks.

Reply:
Squashing commits is part of rebase process, it doesn't require extra rights I think.

Reply:
ok, I did the rebase so I think it's alright now. Thanks.